### PR TITLE
[WI] test

### PR DESCRIPTION
**Pull request by hector-chauran**

### CODE REVIEW

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-test  test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test!

**Comment by hector-chauran** [comment]
Hello, World!

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,15 +5,15 @@ jobs:
...
+          command: exit 0
  
   test:
     docker:
       - image: alpine
     steps:
       - run:
           name: Test
-          command: exit 1
+          command: exit 0
```
Comment (hector-chauran) [inline .circleci/config.yml]: Testing!

**Comment by hector-chauran** [comment]
Hello, World!

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test!

Comment (hector-chauran) [reply .circleci/config.yml]: Great stuff!

Comment (hector-chauran) [reply .circleci/config.yml]: Great stuff! 2

**Comment by hector-test-2** [comment]
Great stuff! 4

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran  test

**Comment by hector-test-2** [comment]
@hector-chauran  test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran  test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test 2

**Comment by hector-test-2** [comment]
@hector-chauran test 3

**Comment by hector-test-2** [comment]
@hector-chauran test 4

File: .circleci/config.yml (hunk 1 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test

Comment (hector-test-2) [reply .circleci/config.yml]: Test

Comment (hector-test-2) [reply .circleci/config.yml]: Test?

Comment (hector-test-2) [reply .circleci/config.yml]: Test!

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test 1

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran Test

**Comment by hector-test-2** [comment]
@hector-chauran Test

**Comment by hector-test-2** [comment]
@hector-chauran Test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test-2

**Comment by hector-test-2** [comment]
@hector-chauran Test!

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-test-2** [comment]
@hector-chauran test 2

**Comment by hector-test-2** [comment]
@hector-chauran test

**Comment by hector-chauran** [comment]
test

**Comment by hector-chauran** [comment]
@hector-test-2 test

**Comment by hector-test-2** [comment]
:slightly_smiling_face:

**Comment by hector-test-2** [comment]
:slightly_smiling_face:

**Comment by hector-test-2** [comment]
:slightly_smiling_face:

**Comment by hector-test-2** [comment]
:slightly_smiling_face:

**Comment by hector-test-2** [comment]
:slightly_smiling_face:

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,15 +5,15 @@ jobs:
...
```
Comment (hector-chauran) [inline .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

**Comment by hector-test-2** [comment]
@hector-chauran test

File: .circleci/config.yml [hunk 3]
```diff
@@ -5,15 +5,15 @@ jobs:
     steps:
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran test

File: .circleci/config.yml [hunk 4]
```diff
@@ -5,15 +5,15 @@ jobs:
...
       - run:
```
Comment (hector-test-2) [inline .circleci/config.yml]: test

File: .circleci/config.yml [hunk 5]
```diff
@@ -5,15 +5,15 @@ jobs:
...
           name: Force failure
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran test

File: .circleci/config.yml [hunk 6]
```diff
@@ -5,15 +5,15 @@ jobs:
...
-          command: exit 1
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran test 2

File: .circleci/config.yml [hunk 7]
```diff
@@ -5,15 +5,15 @@ jobs:
...
+          command: exit 0
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran Test!

File: .circleci/config.yml [hunk 8]
```diff
@@ -5,15 +5,15 @@ jobs:
...
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran test

File: .circleci/config.yml [hunk 9]
```diff
@@ -5,15 +5,15 @@ jobs:
...
  
   test:
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran A

File: .circleci/config.yml [hunk 10]
```diff
@@ -5,15 +5,15 @@ jobs:
...
     docker:
```
Comment (hector-test-2) [inline .circleci/config.yml]: @hector-chauran B

Comment (hector-chauran) [reply .circleci/config.yml]: :)

Comment (hector-chauran) [reply .circleci/config.yml]: @hector-test-2 :slightly_smiling_face:

Comment (hector-chauran) [reply .circleci/config.yml]: response

Comment (hector-chauran) [reply .circleci/config.yml]: asd

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: test

File: .circleci/config.yml [hunk 11]
```diff
@@ -5,15 +5,15 @@ jobs:
...
       - image: alpine
     steps:
       - run:
```
Comment (hector-chauran) [inline .circleci/config.yml]: @hector-test-2 test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: test 2

Comment (hector-chauran) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test 3

Comment (hector-chauran) [reply .circleci/config.yml]: Test 4

Comment (hector-chauran) [reply .circleci/config.yml]: Test 5

Comment (hector-chauran) [reply .circleci/config.yml]: Test 6

Comment (hector-chauran) [reply .circleci/config.yml]: Test 7

Comment (hector-chauran) [reply .circleci/config.yml]: Test 8

Comment (hector-chauran) [reply .circleci/config.yml]: Test 9

Comment (hector-chauran) [reply .circleci/config.yml]: Test 10

Comment (hector-chauran) [reply .circleci/config.yml]: Test 11

Comment (hector-chauran) [reply .circleci/config.yml]: Test 12

Comment (hector-chauran) [reply .circleci/config.yml]: Test 13

Comment (hector-chauran) [reply .circleci/config.yml]: Test 14

Comment (hector-chauran) [reply .circleci/config.yml]: Test 15

Comment (hector-chauran) [reply .circleci/config.yml]: Test 16

Comment (hector-chauran) [reply .circleci/config.yml]: Test 17

Comment (hector-chauran) [reply .circleci/config.yml]: Test 18

Comment (hector-chauran) [reply .circleci/config.yml]: Test 19

Comment (hector-chauran) [reply .circleci/config.yml]: Test 20

Comment (hector-chauran) [reply .circleci/config.yml]: Test 21

Comment (hector-chauran) [reply .circleci/config.yml]: Test 22

Comment (hector-chauran) [reply .circleci/config.yml]: Test 23

Comment (hector-chauran) [reply .circleci/config.yml]: Test 23

Comment (hector-chauran) [reply .circleci/config.yml]: Test 24

Comment (hector-chauran) [reply .circleci/config.yml]: Test 25

Comment (hector-chauran) [reply .circleci/config.yml]: Test 26

Comment (hector-chauran) [reply .circleci/config.yml]: Test 27

Comment (hector-chauran) [reply .circleci/config.yml]: @hector-test @hector-test-2 Test 28

Comment (hector-chauran) [reply .circleci/config.yml]: Test 29

Comment (hector-test) [reply .circleci/config.yml]: test

Comment (hector-chauran) [reply .circleci/config.yml]: Test 30

Comment (hector-chauran) [reply .circleci/config.yml]: Test 31

File: .circleci/config.yml (hunk 4 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test 1-1

File: .circleci/config.yml (hunk 3 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test 1

File: .circleci/config.yml [hunk 12]
```diff
@@ -5,15 +5,15 @@ jobs:
...
           name: Force failure
-          command: exit 1
+          command: exit 0
  
   test:
     docker:
       - image: alpine
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml (hunk 10 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test 3

File: readme [hunk 13]
```diff
@@ -1 +1 @@
-Test repo for Toast!!!!!!!!!!!!!
```
Comment (hector-test-2) [inline readme]: Test 5

**Pull request closed without merging**